certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Impexium",
+    "description": "Impexium is a SaaS based AMS that allows businesses to automate  workflows, including adding activities, contacts, relationships and committee members.",
+    "version": "1.0",
+	"contact": {
+      "name": "Impexium Support",
+      "url": "support.impexium.com",
```

Comment:
This will need to be a complete URL with a scheme. For example, `https://support.impexium.com`